extensions/amp-date-countdown/validator-amp-date-countdown.protoascii

```diff
@@ -36,6 +36,10 @@ tags: {  # <amp-date-countdown>
     value_casei: "minutes"
     value_casei: "seconds"
   }
+  attrs {
+    name: "count-up"
+    value: ""
+  }
   attrs {
     name: "data-count-up"
```

Comment:
Please leave a comment that this is noop in 1.0

Reply:
Great!  Added!